meson.build

```diff
@@ -332,6 +332,9 @@ endif
 if cc.has_header('linux/ethtool.h')
   conf.set('HAVE_ETHTOOL_H', '1')
 endif
+if cc.has_header('linux/bsg.h') and cc.has_header('scsi/scsi_bsg_ufs.h')
```

Comment:
I think technically these should be separate checks, particularly if you follow my other suggestion to drop `HAVE_BSG_H` and make them mandatory for plugin_scsi to enable.